src/components/styles/EventList.css

```diff
@@ -1,3 +1,30 @@
 .event-list tr .accent {
     color: var(--color-accent-eng)
+}
+
+.event-body {
+    padding-bottom: 20px;
+}
+
+.scroll {
```

Comment:
Make events list scrollable.

Side effect: the homepage events list is also scrollable now but the events list will be removed from the homepage anyways.